Clarify 'personal org' concept. (#5890)
circleci/circleci-docs: jekyll/_cci2/orb-author-intro.md

### Register a namespace
{: #register-a-namespace }

Every organization registered on CircleCI is able to claim **one** unique [namespace]({{site.baseurl}}/2.0/orb-concepts/#namespaces). This includes your personal organization and any organization you are a member of. As each organization or user account is limited to a single namespace, in order to register the namespace for an organization you must be the _owner_ of the organization.

> _note: Within CircleCI, we sometimes refer to individual users as "organizations" or "personal organizations", since from our perspective there aren't notable differences. Hence some commands refer to an `org-name`. This can simply be your GitHub username._

Reviewer: ```suggestion
**Note:** We sometimes refer to individual users as "organizations" or "personal organizations", since from CircleCI's perspective there are no notable differences. Some commands refer to an `org-name` for which you can use your GitHub username.
```

Reviewer: Should this be GitHub or Bitbucket username?

Reviewer: Thanks for this @gmemstr ... This is a good interim fix but I'm also going to create a docs ticket for us to ensure we are clear what we mean wherever we use the term organization in docs.
